tests/test_matrix_props/test_kp_norm.py

```diff
@@ -0,0 +1,36 @@
+"""Tests for the S(k)-norm of a matrix."""
+# Tests in this file, follow the discussion of the characteristics of kp_norm at https://qetlab.com/kpNorm
+
+import re
+
+import numpy as np
+import pytest
+
+from toqito.matrix_props import kp_norm, trace_norm
+from toqito.states import bell
+
+
```

Comment:
We could make this a bit more concise using `pytest`:

```suggestion
@pytest.mark.parametrize("vector, k, p, norm_to_compare", [
    # When (k=1, p= Inf)the kp_norm(vector) is the same as the trace norm (the 1-norm).
    (bell(0), 1, np.inf, 1),
    # When p=2 and k is greater than or equal to one of the input matrice's dimensions, the value calculated is the frobenius norm.
    (np.random.rand(5, 4), 4, 2, 2),
])
def test_kp_norm(vector, k, p, norm)
    calculated_kp_norm = kp_norm(vector, k, p)
    assert calculated_kp_norm == np.linalg.norm(vector, norm_to_compare)
```

(Will need to import `pytest` and am not totally sure if this will work. If not, keeping the original is fine).